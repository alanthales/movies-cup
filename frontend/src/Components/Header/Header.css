.header-component {
  color: #fff;
  text-align: center;
  padding: 40px;
}

.header-component h4 {
  color: #333;
  font-size: 90%;
  line-height: 5%;
}