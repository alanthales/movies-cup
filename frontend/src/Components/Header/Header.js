import "./Header.css";
import React from "react";
import Box from "@material-ui/core/Box";
import Grid from "@material-ui/core/Grid";

const Header = ({ title, subtitle }) => (
  <Box bgcolor="primary.main">
    <Grid container justify="center" className="header-component">
      <Grid item xs={8}>
        <h4>CAMPEONATO DE FILMES</h4>
        <h1>{title}</h1>
        <hr style={{ width: "50px" }} />
        <p>{subtitle}</p>
      </Grid>
    </Grid>
  </Box>
);

export default Header;