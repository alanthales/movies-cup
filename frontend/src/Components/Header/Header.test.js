import React from "react";
import Header from "./Header";
import renderer from "react-test-renderer";

test("render Header from given props", () => {
  const component = renderer.create(
    <Header title="Title Test" subtitle="Testing header" />
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});