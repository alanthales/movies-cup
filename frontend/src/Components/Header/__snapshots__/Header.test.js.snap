// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Header from given props 1`] = `
<div
  className="MuiBox-root MuiBox-root-2"
>
  <div
    className="MuiGrid-root header-component MuiGrid-container MuiGrid-justify-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8"
    >
      <h4>
        CAMPEONATO DE FILMES
      </h4>
      <h1>
        Title Test
      </h1>
      <hr
        style={
          Object {
            "width": "50px",
          }
        }
      />
      <p>
        Testing header
      </p>
    </div>
  </div>
</div>
`;
